File: class16/java/AP1611-AprovadoComRecuperacao.java
import java.util.Scanner;
public class AprovadoComRecuperacao {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Nota final: "); double nota = sc.nextDouble();
        System.out.print("Frequência (%): "); double freq = sc.nextDouble();
        if (nota >= 70 && freq >= 75) System.out.println("APROVADO!");
        else if (nota >= 40 && freq >= 75) System.out.println("RECUPERAÇÃO");
        else System.out.println("REPROVADO");
        System.out.println("Nota: " + nota + " | Frequência: " + freq + "%");
        sc.close();
    }
}

File: class16/cpp/AP1611-AprovadoComRecuperacao.cpp
#include <iostream>
int main() {
    double nota, freq;
    std::cout << "Nota final: "; std::cin >> nota;
    std::cout << "Frequência (%): "; std::cin >> freq;
    if (nota >= 70 && freq >= 75) std::cout << "APROVADO!" << std::endl;
    else if (nota >= 40 && freq >= 75) std::cout << "RECUPERAÇÃO" << std::endl;
    else std::cout << "REPROVADO" << std::endl;
    std::cout << "Nota: " << nota << " | Frequência: " << freq << "%" << std::endl;
    return 0;
}